Makefile

```diff
@@ -164,13 +164,28 @@ define docker-up
 	fi
 endef
 
+docker: docker-
```

Comment:
This target fails when I run it, with:

```
cc  -s     -o .o
cc: fatal error: no input files
compilation terminated.
<builtin>: recipe for target '.o' failed
make: *** [.o] Error 4
make: *** [docker-.o] Error 2
```

What's the purpose of this target?